auto_install/install.sh

```diff
@@ -296,6 +311,13 @@ distroCheck(){
 				;;
 			esac
 		;;
+		Alpine)
+		PKG_MANAGER='apk'
+		UPDATE_PKG_CACHE="${PKG_MANAGER} update"
+		PKG_INSTALL="${PKG_MANAGER} --no-cache add"
+		PKG_COUNT="${PKG_MANAGER} --no-cache upgrade -s | grep -sEe ' [0-9]+ packages' | sed -E -e 's/.* ([0-9]+) packages/\1/' || true"
```

Comment:
Looks like the count of installed packages, not outdated packages.

Reply:
`${PKG_COUNT}` is passed to [notifyPackageUpdatesAvailable()](https://github.com/giulio-coa/pivpn/blob/041a820593e1ee8d531ce06d1393cf745ef8ad16/auto_install/install.sh#L449-L464) to inform the user they have out of date packages, but the current command acquires the total number of installed packages.